src/java/org/apache/cassandra/concurrent/Stage.java

```diff
@@ -51,6 +52,7 @@
 public enum Stage
 {
     READ              (false, "ReadStage",             "request",  DatabaseDescriptor::getConcurrentReaders,        DatabaseDescriptor::setConcurrentReaders,        Stage::multiThreadedLowSignalStage),
+    INDEX_READ        (false, "IndexReadStage",        "request",  Stage::getIndexReadStageThreadNum,               null,                                            Stage::multiThreadedLowSignalStage),
```

Comment:
can you make the config match the others (thread it through DD, etc)?

Reply:
on second thought -- let's just keep dumping index reads in to READ stage for now.  If we need a separate stage we can add it later.

Reply:
I say this because we've apparently gone N years with point reads and range reads (and multirange) all on the same stage and it works well enough